boltdesignsystem/bolt
# Fix: add only to twig includes

## Jira

http://vjira2:8080/browse/BDS-2102

## Summary

Fixes certain twig includes without `only` which would cause prop options to trickle down.

## How to test

Run the branch locally and check all Academy Blueprints, make sure they match exactly with their counterparts on `master`.
